docs/theming/class-names-update-280.md

```diff
@@ -0,0 +1,19 @@
+# Class names update in 2.8.0
+
+In WC Blocks 2.8.0, some class names were updated in order so simplify them, fix inconsistencies and make it easier to differentiate frontend components from editor components.
+
+## Replaced classes
+
+Some classes that were introduced in 2.6.0 and 2.7.0 and didn't ship in WooCommerce Core have been replaced by new ones. They can be found in this table:
+
+| Removed | New class name |
+| --- | --- |
+
```

Comment:
Is this incomplete, or was it intentionally left blank?

Reply:
Yes, I left it blank in purpose. This will be filled when we replace more classes according to the plan in #2457. I wanted to keep the scope of this PR small and mainly focus in docs, but after merge I will work on a follow-up to handle all the classes that need to be changed and will fill this list accordingly.